Framework/DataHandling/src/SaveCanSAS1D2.cpp

```diff
@@ -205,7 +205,7 @@ void SaveCanSAS1D2::createSASProcessElement(std::string &sasProcess) {
   if (run.hasProperty("UserFile")) {
     user_file = run.getLogData("UserFile")->value();
   } else {
-    g_log.warning("No user file was found in the input workspace.");
+    g_log.information("No user file was found in the input workspace.");
```

Comment:
Notice is the level at which algorithms log at start/end, I think this will add a lot of noise to that channel as many files are normally saved. Besides, being completely optional, I find this exactly appropriate for information.